source/draw/gpu/opengl/backend/gles3/Gles3Debug.ooc

```diff
@@ -41,6 +41,12 @@ validate: func (message: String) {
 		Debug print(errorMessage)
 		raise(errorMessage)
 	}
+	eglError := eglGetError()
+	if (eglError != EGL_SUCCESS) {
+		errorMessage := "EGL error " + message + ": " + getEglErrorMessage(eglError)
+		Debug print(errorMessage)
+		raise(errorMessage)
```

Comment:
int that case i guess this should work too? :)

`
eglError := eglGetError()
raise(eglError != EGL_SUCCESS, "EGL error " + message + ": " + getEglErrorMessage(eglError))
`